lib/models/command.js

```diff
@@ -186,6 +186,9 @@ let Command = CoreObject.extend({
   onInterrupt() {
     if (this._currentTask) {
       return this._currentTask.onInterrupt();
+    } else {
+      // interrupt all external commands which don't use `runTask()` with an error
```

Comment:
couldn't find a better term for external commands. Alternative was `3rd party commands`